remote_vnc_backend_template.sh

```diff
@@ -18,8 +18,13 @@ create_vnc_password() {
 
 
 create_vncserver_unit_file () {
-	cp /{lib,etc}/systemd/system/vncserver@.service
-	sed -i "s|<USER>|<USERNAME>|g" /etc/systemd/system/vncserver@.service
+	if [ $DISTRO_VERSION = "8" ]; then
+		cp /usr/lib/systemd/user/vncserver@.service /etc/systemd/system/vncserver@.service
```

Comment:
On RHEL8, there may be both `/usr/lib/systemd/user/vncserver@.service` and `/usr/lib/systemd/system/vncserver@.service` (but maybe not in all builds of `tigervnc-server`). While the former one needs the special treatment as shown here, the latter one works with the `sed -i "s|<USER>|<USERNAME>|g" /etc/systemd/system/vncserver@.service` command.

Reply:
@matusmarhefka Do you wish to rework this in a way that if `/usr/lib/systemd/system/vncserver@.service` exists it uses the other sed command? Or is it OK for you to always  use the `/usr/lib/systemd/user/vncserver@.service` ?